test/extended/cli/observe.go

```diff
@@ -68,11 +68,11 @@ var _ = g.Describe("[sig-cli] oc observe", func() {
 		g.By("argument templates")
 		out, err = oc.Run("observe").Args("services", "--once", "--all-namespaces", "--template='{ .spec.clusterIP }'").Output()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(out).To(o.ContainSubstring("172.30.0.1"))
+		o.Expect(out).To(o.Or(o.ContainSubstring("172.30.0.1"), o.ContainSubstring("::1")))
```

Comment:
Does this work? I thought the cluster IP would be something like fd02::982f.

Reply:
Actually I'm trying to find the most common specific IPv6 pattern for all. When I looked at the default kubernetes service;
`{
    "apiVersion": "v1",
    "kind": "Service",
    "metadata": {
        "creationTimestamp": "2021-04-06T06:41:02Z",
        "labels": {
            "component": "apiserver",
            "provider": "kubernetes"
        },
        "name": "kubernetes",
        "namespace": "default",
        "resourceVersion": "231",
        "uid": "58f12afe-f051-4d8c-8af1-6c6285f53791"
    },
    "spec": {
        "clusterIP": "fd02::1",
        "clusterIPs": [
            "fd02::1"
        ],
        "ports": [
            {
                "name": "https",
                "port": 443,
                "protocol": "TCP",
                "targetPort": 6443
            }
        ],
        "sessionAffinity": "None",
        "type": "ClusterIP"
    },
    "status": {
        "loadBalancer": {}
    }
}`

The other option is like the previous commit checking `:`.

Reply:
We could hardcode `fd02::1` like they've hardcoded `172.30.0.1`.  

It appears to make an assumption that it's always the first address in the service network, so if that assumption is OK for IPv4 it should be OK for IPv6 too.